Bottom: d3ec6b8ac4e21608b10cc1493a58f76330c2aad7
Top:    170bd5a990ef76c111a06c974a5d93d8325ac0fd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-27 00:23:23 +0000

hid/gtk: Remove unused Vx2 and Vy2 functions from gtkhid-{main,gdk}.c


---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index b1dc11d..0a2cc59 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -28,21 +28,6 @@ extern HID ghid_hid;
 static int cur_mask = -1;
 static int mask_seq = 0;
 
-/* ------------------------------------------------------------ */
-
-static inline int
-Vx2 (int x)
-{
-  return (x - gport->view_x0) / gport->zoom + 0.5;
-}
-
-static inline int
-Vy2 (int y)
-{
-  return (y - gport->view_y0) / gport->zoom + 0.5;
-}
-
-/* ------------------------------------------------------------ */
 
 typedef struct hid_gc_struct
 {
diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 6b2dd87..9fd781c 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -33,21 +33,6 @@ static void zoom_by (double factor, int x, int y);
 
 int ghid_flip_x = 0, ghid_flip_y = 0;
 
-/* ------------------------------------------------------------ */
-
-static inline int 
-Vx2 (int x)
-{     
-  return (x - gport->view_x0) / gport->zoom + 0.5;
-}       
-        
-static inline int 
-Vy2 (int y)
-{     
-  return (y - gport->view_y0) / gport->zoom + 0.5;
-}       
-
-/* ------------------------------------------------------------ */
 
 static void
 ghid_pan_fixup ()
